### ISSUE TITLE
UI DOESNT PICKUP ARTIFACTS IN LATEST VERSION

### ISSUE THREAD

**Issue by yogeshwaran-98** [mlflow/mlflow#18050]
with mlflow.start_run():
    
    mlflow.log_params(params)

    mlflow.log_metric("accuracy",accuracy)

    signature=infer_signature(X_train,lr.predict(X_train))

    model_info=mlflow.sklearn.log_model(
        sk_model=lr,
        artifact_path="iris_model",
        signature=signature,
        input_example=X_train,
        registered_model_name="tracking-quickstart",

    )

In latest version , when using mlflow.sklearn.log_model which auto generates artifacts , i see a folder starting with m-id nested inside mlartifacts folder , and the ui doesnt pickup the artifacts created like conda.yaml, model.pkl etc 

but when i use the same code in v2.16.2 , it works fine , i can see the artifacts displayed in the ui . What i am missing ??

i run this in jupyterlab locally , and execute the mlflow ui command

**Comment by B-Step62** [comment]
@yogeshwaran-98 Artifact should be available in the model page. You can go to `Versions` tab, open the logged model, then go to the artifact tab.

<img width="933" height="211" alt="Image" src="https://github.com/user-attachments/assets/a4364fa5-b1c6-47c5-9f60-f49d5be445dd" />

<img width="328" height="309" alt="Image" src="https://github.com/user-attachments/assets/678ec18b-dbfc-4e74-bcfc-2f4d1f968867" />

**Comment by yogeshwaran-98** [comment]
Hi @B-Step62 
There is no logged model under experiments -> models , even though i registered it . It is available only under separate model section which does not have a artifact option .

<img width="1643" height="826" alt="Image" src="https://github.com/user-attachments/assets/18caf2f1-bfb5-4714-b71b-f2050e96513a" />

<img width="961" height="392" alt="Image" src="https://github.com/user-attachments/assets/5ef77885-d704-4b4a-8e25-5a6ebbefe827" />

<img width="1201" height="617" alt="Image" src="https://github.com/user-attachments/assets/18d72054-468e-4416-ac27-d6d25f715fae" />

code is same i gave above

**Comment by B-Step62** [comment]
Is the experiment set to "MLFLOW Quickstart2"? Maybe check default experiment to see if the model is logged there instead.

**Comment by yogeshwaran-98** [comment]
Default has nothing 

<img width="1112" height="622" alt="Image" src="https://github.com/user-attachments/assets/fb46c080-b6b9-49d1-83ea-5ddbb3f1b60a" />

<img width="1493" height="819" alt="Image" src="https://github.com/user-attachments/assets/20911c2d-d295-4e01-9af3-91ece7a4babe" />

code:
mlflow.set_tracking_uri(uri="http://127.0.0.1:5000")
mlflow.set_experiment("MLFLOW Quickstart2")

with mlflow.start_run():
    mlflow.log_params(params)
    mlflow.log_metric("accuracy",accuracy)
    signature=infer_signature(X_train,lr.predict(X_train))

    model_info=mlflow.sklearn.log_model(
        sk_model=lr,
        artifact_path="iris_model",
        signature=signature,
        input_example=X_train,
        registered_model_name="tracking-quickstart",

    )

**Comment by rosinaderks** [comment]
I am experiencing the same issue when upgrading from 2.22.2 to 3.3.2. When running code with 2.22.2, we could see the artifacts for each run via Experiments - select run - artifact. When running code with 3.3.2, we changed artifact_path to name as said in the [docs](https://mlflow.org/docs/3.3.2/api_reference/python_api/mlflow.sklearn.html) "artifact_path – Deprecated. Use name instead." We also don't see the option "artifacts" via the Models tab (models - select model - select version)

```
# Log the model
model_info = mlflow.sklearn.log_model(
    sk_model=model, 
    name=model_name, 
    signature=mlflow.models.infer_signature(X_train, model.predict(X_train)),  
    input_example=X_train, 
)
```
See attached images for the difference.
<img width="879" height="432" alt="Image" src="https://github.com/user-attachments/assets/ddbe04da-062c-4985-89fd-456cde5534ef" />

<img width="705" height="267" alt="Image" src="https://github.com/user-attachments/assets/f1c604fd-1e09-4c1a-936a-f066da212323" />

<img width="1899" height="732" alt="Image" src="https://github.com/user-attachments/assets/ace96f61-f602-4aa2-b7b5-71a24415cc9f" />

**Comment by B-Step62** [comment]
> We also don't see the option "artifacts" via the Models tab (models - select model - select version)

The "model" tab I referred above is not that one. The "model" selected under the experiment. Do you see any model logged there?

<img width="1866" height="422" alt="Image" src="https://github.com/user-attachments/assets/3dff0b12-02cc-4b61-a269-676b0af2a92a" />

**Comment by rosinaderks** [comment]
Thanks for the clarification, I can see the model artifacts there! I do wish to mention that I don't understand why it is moved to a tab which says "Experimental", while before it was not experimental. This is why I did not bother to look under this tab (human ways :)). 

<img width="638" height="109" alt="Image" src="https://github.com/user-attachments/assets/ab22818f-0f37-4cb6-bb37-2bcfb85d91ac" />
